containers/compose.elasticsearch.yaml

```diff
@@ -2,18 +2,23 @@ version: "3.8"
 
 services:
   es01:
+    hostname: es01
     image: docker.elastic.co/elasticsearch/elasticsearch:${STACK_VERSION}
     volumes:
       - type: bind
-        source: ${NGINX_CONFIG}/certificates
+        source: ${HOST_CERTBOT_PATH_CONF}/archive/${CERTBOT_DOMAIN}
```

Comment:
> ${HOST_CERTBOT_PATH_CONF}/archive/${CERTBOT_DOMAIN}

Maybe here you need to bind to 'live' instead of 'archive':
`${HOST_CERTBOT_PATH_CONF}/live/${CERTBOT_DOMAIN}/cert.pem`.
Because the 'archive' directory will store multiple versions of the certificate; for example, cert1.pem, cert2.pem, cert3.pem, and the 'live' directory will be a shortcut to the certificate of the updated version:

`cert.pem` -> `../../archive/es.literev.com/cert2.pem`